samples/tutorial/README.md

```diff
@@ -0,0 +1,31 @@
+# Tutorial
+
+## Overview
+
+Oh hi! Looks like you want build some software with Workflows! It's a bit different from traditional
+Android development, so let's go through building a simple little TODO app to get the basics down.
+
+## Layout
+
+The project has both a starting point, as well as an example of the completed tutorial.
+
+To help with the setup, we have created a few helpers:
+
+- `tutorial-views`: A set of 3 views for the 3 screens we will be building, `Welcome`, `TodoList`,
+  and `TodoEdit`.
+- `tutorial-base`: This is the starting point to build out the tutorial. It contains view
+  controllers that host the views from `TutorialViews` to see how they display.
```

Comment:
"view controllers"? Was this from the iOS one?

```suggestion
- `tutorial-base`: This is the starting point to build out the tutorial.
```

Reply:
Good catch, found a few more as well.